docs/integrations/prefect-gcp/index.mdx

```diff
@@ -26,23 +26,41 @@ Upgrade to the latest versions of `prefect` and `prefect-gcp`:
 pip install -U "prefect[gcp]"
 ```
 
-If using BigQuery, Cloud Storage, Secret Manager, or Vertex AI, see [additional installation options](#additional-installation-options).
+If using BigQuery, Cloud Storage, Secret Manager, or Vertex AI, see [additional installation options](#install-extras).
 
+#### Install extras
 To install `prefect-gcp` with all additional capabilities, run the install command above and then run the following command:
 
 ```bash
 pip install "prefect-gcp[all_extras]"
 ```
 
+Or, install extras individually:
+
+```bash
+# Use Cloud Storage
+pip install "prefect-gcp[cloud_storage]"
+
+# Use BigQuery
+pip install "prefect-gcp[bigquery]"
+
+# Use Secret Manager
+pip install "prefect-gcp[secret_manager]"
+
+# Use Vertex AI
+pip install "prefect-gcp[aiplatform]"
+```
+
```

Comment:
could we do a `<CodeGroup>` here for `pip` and `uv` here as suggested here?

https://github.com/PrefectHQ/prefect/pull/17088#discussion_r1951265824